docs/modules/ROOT/pages/explanations/decisions/postgres-monitoring.adoc

```diff
@@ -0,0 +1,109 @@
+= PostgreSQL Managed by VSHN Monitoring
+
+== Problem
+
+We need to provide PostgreSQL Managed by VSHN monitoring checks:
+
+* Availability check
+
+== Requirements for availability check
+
+* PostgreSQL is accessible on network level
+* it's possible to login to PostgreSQL
+* it's possible to execute queries 
+
+== Solutions
+
+PostgreSQL managed by Stackgres:
+
+* it doesn't reuse generated connection string which our users will use, which could lead to situation where our monitoring is green, but users can't login (issues with Service)
+* it's sidecar, issue with exporter would require pod restarting
```

Comment:
```suggestion
* The metrics exporter is a sidecar container, an issue with it would require the Pod to restart and therefore causing downtime to the database
```